addOns/network/src/main/java/org/zaproxy/addon/network/GlobalExclusionsOptions.java

```diff
@@ -188,9 +189,156 @@ protected void updateConfigsImpl(int fileVersion) {
                     setGlobalExclusions(DEFAULT_GLOBAL_EXCLUSIONS);
                     break;
                 }
+            case 1:
+                updateToVersion2();
         }
     }
 
+    private void updateToVersion2() {
+        List<GlobalExclusion> rulesBeingUpdated = getVersion1DefaultRules();
+        List<HierarchicalConfiguration> fields =
+                ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
+        List<String> oldValues = new ArrayList<>(rulesBeingUpdated.size());
+        rulesBeingUpdated.forEach(e -> oldValues.add(e.getValue()));
+        for (HierarchicalConfiguration sub : fields) {
+            try {
+                String value = sub.getString(GLOBAL_EXCLUSION_VALUE_KEY, "");
+                if (oldValues.contains(value)) {
+                    String name = sub.getString(GLOBAL_EXCLUSION_NAME_KEY, "");
+                    // Fix naming mistake
+                    if (name.contains("adx")) {
+                        name = name.replace("adx", "axd");
+                        sub.setProperty(GLOBAL_EXCLUSION_NAME_KEY, name);
+                    }
+                    // Extend image types pattern(s)
+                    if (value.contains("|bmp)")) {
+                        value = value.replace("|bmp)", "|bmp|svg|webp)");
+                        sub.setProperty(GLOBAL_EXCLUSION_VALUE_KEY, value);
+                    }
+                    // Extend audio/video types pattern(s)
+                    if (value.contains("|og[gav])")) {
+                        value = value.replace("|og[gav])", "|og[gav]|webm)");
+                        sub.setProperty(GLOBAL_EXCLUSION_VALUE_KEY, value);
+                    }
+                    // Make them case insensitive
+                    value = "(?i)" + value;
+                    sub.setProperty(GLOBAL_EXCLUSION_VALUE_KEY, value);
+                }
+            } catch (ConversionException e) {
+                LOGGER.warn("An error occurred while reading a global exclusion:", e);
+            }
+        }
+    }
+
+    private static List<GlobalExclusion> getVersion1DefaultRules() {
+        return List.of(
```

Comment:
This should be in `updateToVersion2` we also need only the values.

Reply:
I had it there originally but then I felt it distracted from the actual ummm utility of the method.

Reply:
I'd still inline it.

Reply:
Done